static/app/components/feedback/feedbackItem/feedbackScreenshot.tsx

```diff
@@ -1,107 +1,57 @@
-import type {ReactEventHandler} from 'react';
-import {Fragment, useState} from 'react';
+import {useState} from 'react';
 import styled from '@emotion/styled';
 
 import {Role} from 'sentry/components/acl/role';
-import {Button} from 'sentry/components/button';
 import ImageVisualization from 'sentry/components/events/eventTagsAndScreenshot/screenshot/imageVisualization';
 import LoadingIndicator from 'sentry/components/loadingIndicator';
 import Panel from 'sentry/components/panels/panel';
-import PanelBody from 'sentry/components/panels/panelBody';
-import PanelHeader from 'sentry/components/panels/panelHeader';
-import {IconChevron} from 'sentry/icons';
-import {t, tct} from 'sentry/locale';
-import {space} from 'sentry/styles/space';
-import type {Event, EventAttachment, Organization, Project} from 'sentry/types';
+import type {EventAttachment, Organization, Project} from 'sentry/types';
 
 type Props = {
-  eventId: Event['id'];
-  openVisualizationModal: (eventAttachment: EventAttachment) => void;
+  onClick: () => void;
   organization: Organization;
   projectSlug: Project['slug'];
   screenshot: EventAttachment;
-  screenshotInFocus: number;
```

Comment:
deleting all the extra stuff !
![gif](https://media0.giphy.com/media/6krtoA3nZ23uHOIMwF/giphy.gif?cid=6c09b952pdrtphvp642c807u5fq4ho54a8vc231fsj03fjgc&ep=v1_internal_gif_by_id&rid=giphy.gif&ct=v)